src/transformers/models/conditional_detr/modeling_conditional_detr.py

```diff
@@ -499,10 +499,11 @@ def build_position_encoding(config):
 
 
 # function to generate sine positional embedding for 2d coordinates
-def gen_sine_position_embeddings(pos_tensor):
+def gen_sine_position_embeddings(pos_tensor, d_model):
```

Comment:
This is because 128 here corresponds to a hard-coded hidden size of 256, the change allows a smaller hidden size.